### PR TITLE
rtmp-services: Update Switchboard Live servers

### PR DESCRIPTION
**Pull request by CiriousJoker**

### CODE REVIEW

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -1262,33 +1262,73 @@
...
+                },
+                {
+                    "name": "Asia South (Mumbai, IN)",
+                    "url": "rtmp://ingest-as-south.a.switchboard.zone/live"
                 }
-            ]
+            ],
+            "recommended": {
+                "keyint": 2,
+                "profile": "main",
```
Comment (notr1ch) [inline plugins/rtmp-services/data/services.json:1328]: Why is H264 main profile recommended?

Comment (CiriousJoker) [reply plugins/rtmp-services/data/services.json:1328]: This is not a restriction in any way, it's just the profile that ensures the best compatibility for all our users.

Comment (dodgepong) [reply plugins/rtmp-services/data/services.json:1328]: Unless users explicitly disable it, OBS will enforce all settings listed in the recommended section. This is "profile", not "minimum profile". Only set this to `main` if you prefer that all OBS users stream to your service with the `main` profile.

Comment (CiriousJoker) [reply plugins/rtmp-services/data/services.json:1328]: Oh, we misunderstood that setting then. In that case we don't need it. Can you remove it before merging or should I create a new pull request?

Comment (notr1ch) [reply plugins/rtmp-services/data/services.json:1328]: You can change it and force push to the same branch the PR was made from and the PR will be updated.

Comment (CiriousJoker) [reply plugins/rtmp-services/data/services.json]: Fixed it

Comment (notr1ch) [reply plugins/rtmp-services/data/services.json]: You will need to squash the commits and force-push, there should only be a single commit for a service change.

Comment (CiriousJoker) [reply plugins/rtmp-services/data/services.json]: I ended up creating a new PR instead.

**Comment by CiriousJoker** [comment]
Closing in favor of 2813

**Pull request closed without merging**